### PR TITLE
Fix cut off label for clipboard max size on GUI Server window

### LINKED ISSUE #1644: Minor UI issue
### What happened?

The [Max Clipboard sharing size] part got weird box with truncated numbers up/down button.

See screenshot, I can't explain better.

Screenshot_20230512_085927

### Version

v2.4.0

### Git commit hash (if applicable)

_No response_

### If applicable, where did you install Input Leap from?

_No response_

### What OSes are you seeing the problem on? (Check all that apply)

Linux

### What OS versions are you using?

Kubuntu 23.04

### Relevant log output

_No response_

### Any other information

_No response_

### PR DESCRIPTION
**Pull request by sithlord48**
Fixes #1644.

Screenshot_20231024_222732

## Contributor Checklist:

* [x] I have created a file in the `doc/newsfragments` directory *IF* it is a
      user-visible change (and make sure to read the `README.md` in that directory)

### CODE REVIEW

File: src/gui/src/ServerConfigDialogBase.ui [hunk 1]
```diff
@@ -17,7 +17,7 @@
    <item>
     <widget class="QTabWidget" name="m_pTabWidget">
      <property name="currentIndex">
-      <number>2</number>
+      <number>0</number>
      </property>
```
Comment (shymega) [inline src/gui/src/ServerConfigDialogBase.ui:21]: Why was this changed?

File: src/gui/src/ServerConfigDialogBase.ui [hunk 2]
```diff
@@ -427,50 +427,46 @@ Double click on a screen to edit its settings.</string>
...
+               </property>
+               <property name="value">
+                <number>3000000</number>
+               </property>
+              </widget>
+             </item>
+            </layout>
            </widget>
           </item>
           <item row="3" column="0">
```
Comment (shymega) [inline src/gui/src/ServerConfigDialogBase.ui:472]: Seems logical.

**Review by shymega — COMMENTED** [review]
Few Qs.

**Comment by shymega** [comment]
Waiting for tests to pass, but we will be making a set of contribution guidelines and the commit messages for this PR need to be improved. In particular, reasons for changes, and justification. Grammar & spelling is also good to have right.

File: src/gui/src/ServerConfigDialogBase.ui (hunk 1 above)
Comment (sithlord48) [reply src/gui/src/ServerConfigDialogBase.ui:21]: The dialog was opening on the incorrectly opening with Advanced tab and not the screens one.

**Comment by shymega** [comment]
The PR also needs a newsfragment. Please make a `.bugfix` extension newsfragment.

File: src/gui/src/ServerConfigDialogBase.ui (hunk 1 above)
Comment (shymega) [reply src/gui/src/ServerConfigDialogBase.ui:21]: Ah, okay. Can you split this change into a new PR, please? That way we can isolate different changes.

**Review by shymega — CHANGES_REQUESTED** [review]
This change should be in a separate PR.

**Comment by shymega** [comment]
I don't think I've considered a usecase for bug fixes that fix specific issues. Let's consult @p12tic here, who introduced newsfragments.

@p12tic How would you name newsfragments that fix specific issue IDs?

**Comment by shymega** [comment]
The other thing is, the commit message should be adjusted to have 'Fixes [....]' on a newline - not in the main commit subject header.

**Comment by sithlord48** [comment]
Removed the starting tab fix, will do in another PR

> The other thing is, the commit message should be adjusted to have 'Fixes [....]' on a newline - not in the main commit subject header.

Done

**Comment by shymega** [comment]
OK, so now we just need to wait for @p12tic's input regarding the newsfragment, then merge is the next step.

**Comment by shymega** [comment]
I've just approved the workflow, so we'll be able to assess on that factor too.

File: src/gui/src/ServerConfigDialogBase.ui [hunk 3]
```diff
@@ -427,50 +427,46 @@ Double click on a screen to edit its settings.</string>
...
+               </property>
+               <property name="singleStep">
+                <number>1000</number>
+               </property>
+               <property name="value">
+                <number>3000000</number>
+               </property>
+              </widget>
+             </item>
+            </layout>
```
Comment (shymega) [inline src/gui/src/ServerConfigDialogBase.ui:469]: This looks fine, but does it work with Qt 5?

File: doc/newsfragments/fix1644.bugfix [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Fixed spacing for max clipboard input on in the advanced server settings (https://github.com/input-leap/input-leap/issues/1644).
+
```
Comment (shymega) [inline doc/newsfragments/fix1644.bugfix:2]: Just need to wait for @p12tic's input here.

File: src/gui/src/ServerConfigDialogBase.ui (hunk 3 above)
Comment (sithlord48) [reply src/gui/src/ServerConfigDialogBase.ui:469]: It should work just fine with both Qt5 and Qt6

Comment (shymega) [reply src/gui/src/ServerConfigDialogBase.ui:469]: Awesome. LGTM.

**Comment by shymega** [comment]
@sithlord48 Please, don't keep requesting my review. There's nothing I can do. I have pinged @p12tic for input on the newsfragment, but for now, we'll need to wait.

It's not an urgent PR. We can put this on the back burner for now.

**Comment by sithlord48** [comment]
> @sithlord48 Please, don't keep requesting my review. There's nothing I can do. I have pinged @p12tic for input on the newsfragment, but for now, we'll need to wait.

I requested re review beause you have a change request on something that has been changed and was trying to get that cleared.

**Comment by shymega** [comment]
> > @sithlord48 Please, don't keep requesting my review. There's nothing I can do. I have pinged @p12tic for input on the newsfragment, but for now, we'll need to wait.
> 
> I requested re review beause you have a change request on something that has been changed and was trying to get that cleared.

My apologies, I didn't see the change. I still can't see any pushes, so that's why I responded with that response. Did you force-push? I'm looking now.

**Review by shymega — APPROVED** [review]
LGTM.
